config/localization/en/app.json

```diff
@@ -733,15 +733,17 @@
     "AFFILIATES": {
       "AFFILIATES_PROGRAM": "Affiliates Program",
       "AFFILIATE_LINK": "Affiliate Link",
-      "EARN_FOR_EACH_TRADER": "Earn up to ${AMOUNT_USD}/mo for each new trader",
+      "AFFILIATE_LINK_REQUIREMENT": "Affiliate Link will activate at ${AMOUNT_USD} volume",
+      "BECOME_A_VIP": "Become a VIP",
+      "EARN_FOR_EACH_TRADER": "Earn up to ${AMOUNT_USD} per month for each new trader",
       "EARN_FEES": "Earn fees",
       "REFER_FOR_DISCOUNTS": "Refer a friend and they can receive up to ${AMOUNT_USD} in discounts.",
-      "REFER_FOR_DISCOUNTS_FIRST_ORDER": "Receive up to ${AMOUNT_USD} in fee discounts after placing your first order!",
-      "EARCH_FOR_EACH_TRADER_REFER_FOR_DISCOUNTS": "Earn up to ${AMOUNT_PER_MONTH}/mo for each new trader you refer. Your referral can save up to ${AMOUNT_DISCOUNT} in trading fees. {LEARN_MORE_LINK}",
+      "REFER_FOR_DISCOUNTS_FIRST_ORDER": "Receive up to {AMOUNT_USD} in fee discounts after placing your first order!",
```

Comment:
hmm I notice we have a slight inconsistency between when we use ${AMOUNT_USD} vs {AMOUNT_USD} (I assume the caller is either passing in $ vs not based on formatting) - should we update to be more consistent?

Reply:
oh yea, this one in particular does not have $ because the design has this number and $ a different color 😅